pyproject.toml

```diff
@@ -39,9 +43,15 @@ exclude = [
     "**/__pycache__"
 ]
 reportMissingImports = true
-typeCheckingMode = "strict"
+typeCheckingMode = "basic"
 
 
 [tool.ruff]
 # See https://beta.ruff.rs/docs/configuration/
 select = ["E", "F", "I001"]
+
+
+[tool.pytest.ini_options]
+markers = [
+    "integration: marks test as integration tests (select with '-m integration')",
+]
```

Comment:
You can't have this toml section twice, copy the `markers` definition up to around line 37 after the `testpaths`